src/pages/ScreeningsPage.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="movies">
+    <BreadCrumbs :steps="steps" :back-route="{ name: 'Home' }" />
+    <div class="movies__screenings">
+      <Screenings title-variation="default" />
+    </div>
+  </div>
+</template>
+<script>
+import BreadCrumbs from "../components/common/BreadCrumbs.vue";
+import Screenings from "../components/HomePage/Screenings.vue";
+import { useMovieStore } from "@/stores/movies";
+import { onMounted } from "vue";
+import { storeToRefs } from "pinia";
+export default {
+  components: {
+    BreadCrumbs,
+    Screenings,
+  },
+  setup() {
+    const steps = [{ text: "Screenings", isLink: false }];
```

Comment:
Considering that these steps only deal with breadcrumbs, how about naming this to `breadcrumbSteps` ? 😄

Reply:
Definitely more explicit